index.rst

```diff
@@ -1,32 +1,24 @@
-:tocdepth: 1
+########################################################################################
+Investigation into vibrations in the M1M3 surrogate on the TMA due to the Fan Coil Units
+########################################################################################
 
-.. sectnum::
+.. abstract::
 
-.. Metadata such as the title, authors, and description are set in metadata.yaml
+   This technote uses data from the VMS, dc accelerometers and IMS to estimate the effect of the Fan Coil Units (FCUs) are likely to have on image quality
 
-.. TODO: Delete the note below before merging new content to the main branch.
 
-.. note::
+Introduction
+============
 
-   **This technote is a work-in-progress.**
+This technote uses data from the VMS, dc accelerometers and IMS to estimate the effect of the Fan Coil Units (FCUs) are likely to have on image quality.
+Primarily we use the Vibration Monitoring System (VMS) attached to M1M3 (for more information see the 
+`VMS confluence page <https://confluence.lsstcorp.org/pages/viewpage.action?pageId=156502157>`_).
+The description of the FCU is avaiable in 
+`SPIE 77331E <https://doi.org/10.1117/12.857438>`_.
 
-Abstract
-========
-
-.. Assuming units of VMS = mg
-.. -displacement for total_1
-..  FCU on = 41.55 μm
-..  FCU off = 2.30 μm
-.. -displacement for total_2
-..  FCU on = 11.56 μm
-..  FCU off = 2.21 μm
-.. -displacement for total_3
-..  FCU on = 46.05 μm
-..  FCU off = 1.94 μm
-
-
-This technote uses data from the VMS, dc accelerometers and IMS to estimate the effect of the Fan Coil Units (FCUs) are likely to have on image quality
-Primarily we use the Vibration Monitoring System (VMS) of the M1M3 for more information see the VMS confluence page (link here).
+The VMS data are stored in a single hdf5 data file (/sdf/data/rubin/shared/mtm1m3_test_files/vms_data/2023/12/M1M3-2023-12-07T00:00.hdf) acquired on 2023/12/07. A description of the 
```

Comment:
Could you put single-back-quotes before and after the path to convert it into code-style? This formatting can improve readability. Something like this:
`/sdf/data/rubin/shared/mtm1m3_test_files/vms_data/2023/12/M1M3-2023-12-07T00:00.hdf`

Reply:
done